package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "openaq-quality-checker",
-  "version": "1.0.0",
+  "name": "openaq-quality-check",
```

Comment:
It's mildly confusing for the name referenced in the readme to be "OpenAQ Quality Checks" and the package name be openaq-quality-check. I can imagine getting confused/forgetting if it's supposed to be singular or plural.

Not really a blocker for this PR, though.

Reply:
Oh but the repo name should definitely be the same as what's in the package.json, and ideally the same as the name of the npm module.

Reply:
@sethvincent I agree and this name is hopefully temporary - I mistakenly published `openaq-quailty-checks` to npm under the wrong npm user (cmsgov) so once I get the owner of that npm user to force unpublish `openaq-quality-checks` I might be able to re-publish under the desired name.